README.md

```diff
@@ -173,9 +175,11 @@ let fc = g:firenvim_config['localSettings']
 let fc['https?://[^/]+\.co\.uk/'] = { 'takeover': 'never', 'priority': 1 }
 ```
 
-From now on, localSettings examples will use the `let fc[...] = ...` shorthand, assuming that you have defined a `g:firenvim_config` object and that you have a line like `let fc = g:firenvim_config['localSettings']` in your config.
+Note that firenvim will always be broken on some sites, e.g. [Google Docs](https://github.com/glacambre/firenvim/issues/630). If you find a site where firenvim isn't working correctly, first search through the [open issues](https://github.com/glacambre/firenvim/issues) and if there is none [report it](https://github.com/glacambre/firenvim/issues/new) to check whether it [might be fixable](https://github.com/glacambre/firenvim/issues/607). There is a community-maintained [site in the wiki](https://github.com/glacambre/firenvim/wiki/Broken-Websites) with configuration recommendations to minimize breakage - feel free to contribute!
```

Comment:
I think this piece of information doesn't belong here as it does not help understand how Firenvim is configured. Could you find another place in the Readme for this?

Reply:
Yeah, I'll have to see. I'm already stretching my time commitments with these README edits, so I can't overdo it ;)